docs/blogs/posts/2024/2024-01-parakeet.md

```diff
@@ -0,0 +1,103 @@
+* **Permissive license:** Released under CC-BY-4.0 license, model checkpoints can be used in any commercial application.
+
+Parakeet is a major step forward in the evolution of conversational AI. Its exceptional accuracy, coupled with the flexibility and ease of use offered by NeMo, empowers developers to create more natural and intuitive voice-powered applications. The possibilities are endless, from enhancing the accuracy of virtual assistants to enabling seamless real-time communication.. 
+
+The Parakeet family of models achieves state-of-the-art numbers on the [HuggingFace Leaderboard](https://huggingface.co/spaces/hf-audio/open_asr_leaderboard). Users can try out the [parakeet-rnnt-1.1b](https://huggingface.co/nvidia/parakeet-rnnt-1.1b) firsthand at the [Gradio demo](https://huggingface.co/spaces/nvidia/parakeet-rnnt-1.1b). To access the model locally and explore the toolkit, visit the [NVIDIA NeMo Github page](https://github.com/NVIDIA/NeMo). 
+
+## Architecture Details
+
+Parakeet models are based on the [Fast Conformer architecture published in ASRU 2023](https://arxiv.org/abs/2305.05084). Fast Conformer is an optimized version of the Conformer model with 8x depthwise-separable convolutional downsampling, modified Convolution kernel size, and an efficient subsampling module. Additionally it supports inference on very long audio segments (up to 11 hours of speech) on an A100 80GB card using Local Attention. The model is trained end-to-end using the Transducer decoder (RNNT) or Connectionist Temporal Classification decoder. For further details on long audio inference, please refer to the ICASSP 2024 paper “[Investigating End-to-End ASR Architectures for Long Form Audio Transcription](https://arxiv.org/abs/2309.09950)”.
+
+<figure markdown>
+  ![Parakeet architecture](https://github.com/NVIDIA/NeMo/releases/download/v1.21.0/asset-post-2024-01-03-parakeet_arch.png)
+  <figcaption><b>Figure 1.</b> <i> Fast Conformer Architecture shows blocks of downsampling, conformer encoder blocks with limited context attention (LCA), and global token (GT).</i></figcaption>
+</figure>
+
+## Usage
+
+NVIDIA NeMo can be installed as a pip package as shown below. Cython and PyTorch (2.0 and above) should be installed before attempting to install NeMo Toolkit.
+
+Then simply use 
+```bash 
+pip install nemo_toolkit['asr']
+```
+
+Once installed, you can evaluate a list of audio files as follows:
+```python
+import nemo.collections.asr as nemo_asr
+asr_model = nemo_asr.models.ASRModel.from_pretrained(model_name="nvidia/parakeet-rnnt-1.1b")
+transcript = asr_model.transcribe(["some_audio_file.wav"])
+```
+
+## Long-form Speech Inference
+
+Once you have a Fast Conformer model loaded, can easily modify the attention type to limited context attention after building the model. Additionally, you can also apply audio chunking for the subsampling module to perform inference on huge audio files!
```

Comment:
Can you add a sentence that published model was trained with full attention and using limited context will affect the accuracy.